release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -726,6 +731,12 @@ In the following tables, features are marked with the following statuses:
 
 The Bare Metal Event Relay Operator is deprecated. The ability to monitor bare-metal hosts by using the Bare Metal Event Relay Operator will be removed in a future {product-title} release.
 
+==== Dedicated service monitors for core platform monitoring
+With this release, the dedicated service monitors feature for core platform monitoring is deprecated.
+The ability to enable dedicated service monitors by configuring the `dedicatedServiceMonitors` setting in the `cluster-monitoring-config` config map object in the `openshift-monitoring` namespace will be removed in a future {product-title} release.
+As a replacement, the new `trackTimestampsStaleness` setting replicates the behavior of the deprecated setting, thereby helping to ensure thats alerts and time aggregations are accurate. 
+This new setting is enabled by default and requires no further configuration.
```

Comment:
`trackTimestampsStaleness` is actually the Prometheus level feature we enable. This is not user visible, instead user no longer need to do anything to get the benefits of the now deprecated feature.
```suggestion
Prometheus was improved in order to ensure that alerts and time aggregations are accurate. This improvement is active out of the box and makes the dedicated service monitors feature obsolete. 
```

Reply:
Revised per your suggested edits.